graphql/resolve/query_rewriter.go

```diff
@@ -1567,7 +1573,18 @@ func idFilter(filter map[string]interface{}, idField schema.FieldDefinition) []u
 	if idsFilter == nil {
 		return nil
 	}
-	idsSlice := idsFilter.([]interface{})
```

Comment:
This was panic-ing if the type coming in from the wild was a simple string—which I'd say is the typical scenario for IDs when encoded in JWTs (taken from some session ID or something).